tests/manage/z_cluster/test_must_gather.py

```diff
@@ -70,10 +77,39 @@ def finalizer():
 
         request.addfinalizer(finalizer)
 
+    @pytest.mark.polarion_id("OCS-1583")
     def test_must_gather(self):
         """
         Tests functionality of: oc adm must-gather
 
+        """
+        self.collect_must_gather()
+
+    @pytest.mark.polarion_id("OCS-2328")
+    def test_must_gather_worker_node_down(self, nodes):
```

Comment:
this class has `tier1` decorator. This test should not be part of tier1 but rather tier4 IMO.

Reply:
`test_must_gather_worker_node_down` -> tier4a
`test_must_gather` -> tier1

Reply:
+1